### ISSUE TITLE
Broken captions

### ISSUE THREAD

**Issue by stlankes** [flavio20002/typst-orange-template#22]
Hello,

I use your template and add  a figure as followed:

```
#figure(
  image("../images/ken-and-den.jpg", width: 60%),
  caption: [Ken Thompson (sitting) and Denis Ritchie (standing) in front of a PDP-11 #cite(<Den_and_ken>)],
) <Ken_and_den>
```

The figure is now part of the book chapter. But in the section _List of Figures_ the caption isn't shown correctly. Here I see just following text:

```
1.1 Ken Thompson . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 10
```

Do you know the reason?

**Comment by flavio20002** [comment]
Hi, I've understood the problem. I will fix it in the next release of the package.

**Comment by flavio20002** [comment]
Fixed in "0.3.0".
